server/api/batch_inventory.py

```diff
@@ -557,21 +579,21 @@ def download_batch_inventory_batch_tallies(
             "Batch inventory must be signed off before downloading batch tallies."
         )
 
-    assert len(list(jurisdiction.contests)) == 1
-    contest = list(jurisdiction.contests)[0]
+    contest_choice_csv_headers = construct_contest_choice_csv_headers(jurisdiction)
 
     csv_io = io.StringIO()
     batch_tallies = csv.writer(csv_io)
 
-    batch_tallies.writerow(["Batch Name"] + [choice.name for choice in contest.choices])
+    batch_tallies.writerow(["Batch Name", *contest_choice_csv_headers.values()])
```

Comment:
As of Python 3.7 (we're using 3.8), dictionaries are ordered. So calling `.values()` here and `.keys()` later will return values and keys in a corresponding order